bantam-service/src/Bantam/Service/Resource/Fight.hs

```diff
@@ -90,10 +90,10 @@ lemmasResource fightService fightId email req = do
       return $
         halt HTTP.status405 Empty
 
-currentLemmaResource :: MonadIO m => Fight m -> FightId -> Resource m
-currentLemmaResource fightService fightId _req = do
+currentLemmaResource :: (Functor m, MonadIO m) => Fight m -> FightId -> Resource m
+currentLemmaResource fightService fightId req = do
   lemmaId <- notFound $ currentLemma fightService fightId
-  return $ redirect lemmaPath (fightId, lemmaId)
+  lemmaReadResource fightService fightId lemmaId req
```

Comment:
Note you won't be able to edit the current lemma (if you're the owner) now from this URL, which I think makes sense.